### PR TITLE
Fix "dev" mode in CT api-client

### PR DESCRIPTION
**Pull request by filipsobol**

### CODE REVIEW

**Review by Fifciu — APPROVED** [review]
It looks pretty fine for me. However, I wonder, why did you get rid of `pkg.server`? Wouldn't it be nice to have all knowledge about built files in package.json?

**Review by filrak — APPROVED** [review]

**Comment by filipsobol** [comment]
> It looks pretty fine for me. However, I wonder, why did you get rid of `pkg.server`? Wouldn't it be nice to have all knowledge about built files in package.json?

This is a made-up field that doesn't normally exist in `package.json` unlike `main`, `module`, and `types`. It was only needed to provide an entry path for Rollup, so I decided to remove it. Let me know if you think it should stay.

File: packages/commercetools/api-client/rollup.config.js [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+const main = generateBaseConfig(pkg);
+main.plugins.push(graphql());
 
-export default {
+const server = {
   input: 'src/index.server.ts',
   output: [
     {
-      file: pkg.server,
+      file: 'server/index.js',
```
Comment (andrzejewsky) [inline packages/commercetools/api-client/rollup.config.js:13]: why?

Comment (filipsobol) [reply packages/commercetools/api-client/rollup.config.js]: Reverted

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**